src/equations/compressible_euler_multicomponent_2d.jl

```diff
@@ -82,13 +72,16 @@ function initial_condition_convergence_test(x, t, equations::CompressibleEulerMu
   v2      = 1.0
 
   rho     = ini
-  rho1    = 0.2 * rho
-  rho2    = 0.8 * rho  
-  rho_v1  = rho * v1
-  rho_v2  = rho * v2
-  rho_e   = rho^2
+  # Multiple Components
+  prim_rho  = SVector{NCOMP, eltype(gamma)}(2^(i-1) * (1-2)/(1-2^NCOMP) * rho for i = 1:NCOMP)
```

Comment:
That: `@inline Base.real(::CompressibleEulerMulticomponentEquations2D{NCOMP, NVARS, RealT}) where {NCOMP, NVARS, RealT} = RealT` was really helpful, thank you!

I'm not quite sure if the helper functions are really necessary since they won't be used 'that' often and are not considerably shorter.

Reply:
done in 657c2b2